### ISSUE TITLE
How to use this SDK?

### ISSUE THREAD

**Issue by colthreepv** [tuya/tuya-connector-nodejs#1]
At the moment the project seems to not be published on NPM, moreover it's a Typescript project, and needs to be built (in `./lib` as I read the config file) to be consumed

How can we help?

**Comment by colthreepv** [comment]
There is. SORRY 😭 
Link to NPM: https://www.npmjs.com/package/@tuya/tuya-connector-nodejs

### Installation

```bash
npm install @tuya/tuya-connector-nodejs

# or
yarn add @tuya/tuya-connector-nodejs
```

**Comment by Erchoc** [comment]
> There is. SORRY 😭
> Link to NPM: https://www.npmjs.com/package/@tuya/tuya-connector-nodejs
> 
> ### Installation
> ```shell
> npm install @tuya/tuya-connector-nodejs
> 
> # or
> yarn add @tuya/tuya-connector-nodejs
> ```

fixed the doc v2.0.2, thanks
